### PR TITLE
Improved snapshot rollback

### LINKED ISSUE #890: [Kura 2.1.0 QA] Snapshot management in web UI
DEN-SNAP-002 from #788 requires snapshot to be rolled back. The UI mentions possibility of a timeout after which manual refresh is necessary.

The issue is that the timeout probably didn't occur - it was definitely not indicated in any way in the UI, and the operation was rather fast - but UI data was not refreshed. ExamplePublisher's configuration remained as it was before the rollback until manual page refresh. And that happened every time I tried.

The same goes for snapshot file upload.

Kura 2.1.0 build 12, Raspberry Pi 3

### PR DESCRIPTION
**Pull request by nicolatimeus**
Closes #890 and #1700

Signed-off-by: nicolatimeus <nicola.timeus@eurotech.com>

### CODE REVIEW

**Comment by nicolatimeus** [comment]
@markoer can you please have a look at the failing test? The code it triggers is no longer there after my changes so I wouldn't know how to fix it.. maybe can we just remove it?

**Comment by markoer** [comment]
If the update I pushed works, the test can stay in there.

**Comment by cdealti** [comment]
@markoer can you please review?

File: kura/test/org.eclipse.kura.core.test/src/main/java/org/eclipse/kura/core/test/ConfigurationServiceTest.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
 import org.eclipse.kura.core.configuration.XmlSnapshotIdResult;
```
Comment (markoer) [inline kura/test/org.eclipse.kura.core.test/src/main/java/org/eclipse/kura/core/test/ConfigurationServiceTest.java:35]: Update the (c) years.

**Review by markoer — APPROVED** [review]
It fixes the issues it promises to fix. Just update the year.

**Pull request merged**